README.md

```diff
@@ -1 +1,161 @@
+
+### Usage
+
+1. Download the files from this repository  
+You can clone this repository using Git or else you can simply download the files as a Zip using the following link:  
+https://github.com/CAFAudit/audit-service-deploy/archive/develop.zip
+
+2. Configure the external parameters if required  
+The following parameters may be set:
+
+    <table>
+      <tr>
+        <th>Environment Variable</th>
+        <th>Default</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>cluster.name</td>
+        <td>docker-cluster</td>
+        <td>A node can only join a cluster when it shares its cluster.name with all the other nodes in the cluster</td>
+      </tr>
+      <tr>
+        <td>bootstrap.memory_lock</td>
+        <td>true</td>
+        <td>It is vitally important to the health of a node that none of the JVM is ever swapped out to disk</td>
+      </tr>
+      <tr>
+        <td>ES_JAVA_OPTS</td>
+        <td>-Xmx256m -Xms256m</td>
+        <td>Environment variable to set heap size <br />e.g. to use 1GB use ES_JAVA_OPTS="-Xms1g -Xmx1g"</td>
+      </tr>
+      <tr>
+        <td>xpack.security.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack security</td>
+      </tr>
+      <tr>
+        <td>xpack.monitoring.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack monitoring</td>
+      </tr>
+      <tr>
+        <td>xpack.graph.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack graph</td>
+      </tr>
+      <tr>
+        <td>xpack.watcher.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack watcher</td>
+      </tr>
+    </table>
+
+	For more information on X-Pack:
+	* https://www.elastic.co/guide/en/x-pack/current/xpack-settings.html
+	* https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling
+
+3. Deploy the services  
+First navigate to the folder where you have downloaded the files to and then run one of the following commands, depending on whether you are using Docker Compose or Docker Stack:
+
+    <table>
+      <tr>
+        <td><b>Docker Compose</b></td>
+        <td>
+			docker-compose up  <small>(docker-compose defaults to use a file called <i><b>docker-compose.yml</b></i>)</small><br />
+			docker-compose -f docker-compose.yml up <small>(An alternative filename can be provided using the <i><b>-f</b></i> flag</small> <br />
+			docker-compose -f docker-compose.yml up -d <small>(<i><b>-d</b></i> flag is for "detached mode" i.e. run containers in the background)</small>
+		</td>
+      </tr>
+      <tr>
+        <td><b>Docker Stack</b></td>
+        <td>docker stack deploy --compose-file=docker-compose.yml auditservicedemo</td>
+      </tr>
+    </table>
+
+4. Check the Health of Elasticsearch  
+The health of the Elasticsearch container and / or cluster can be inspected by issuing the following command:    
+	`curl http://127.0.0.1:9200/_cat/health`    
```

Comment:
I don't know if we should assume that the Docker Host is running on `127.0.0.1`.  Also, if port `9200` is exposed, as it must be for this command to work, then it should have been defined as an external parameter and listed in step 2, instead of the things that are listed there.